java/src/test/java/ai/rapids/cudf/BinaryOpTest.java

```diff
@@ -1098,4 +1099,17 @@ public void testLogBase2() {
       assertColumnsAreEqual(expected, answer, "log2");
     }
   }
+
+  @Test
+  public void testArctan2() {
+    Double[] xValues = TestUtils.getDoubles(20342309423423L, 10, false);
+    Double[] yValues = TestUtils.getDoubles(33244345234423L, 10, false);
+    try (ColumnVector y = ColumnVector.fromBoxedDoubles(yValues);
+         ColumnVector x = ColumnVector.fromBoxedDoubles(xValues);
+         ColumnVector result = y.arctan2(x);
+         ColumnVector expected = ColumnVector.fromDoubles(IntStream.range(0,xValues.length)
+             .mapToDouble(n -> Math.atan2(yValues[n], xValues[n])).toArray())) {
+      assertColumnsAreEqual(expected, result);
```

Comment:
The C++ version was using an approximate equals for this operator. To me, this means that either we are not picking enough values or we shouldn't bother with random number selection because the point of these tests should mostly be to verify the plumbing.

Reply:
Hmm... The result being returned by cudf match exactly with the values in Java even after using 50 values. I understand your concern but I don't understand why. @sriramch can you shed some light on this?

```
[Infinity, Infinity, 0.0, 8.131693407815494E307, 1.2460157142072596E308, NaN, 0.0, 4.559269070904487E307, NaN, NaN, Infinity, Infinity, NaN, 1.7956896655964089E307, NaN, NaN, 4.9E-324, Infinity, 0.0, -Infinity, -Infinity, 4.9E-324, NaN, NaN, NaN, 0.0, 4.9E-324, 0.0, NaN, 1.7372014381085476E308, -Infinity, 4.9E-324, -Infinity, NaN, NaN, 0.0, 9.490595879365074E307, NaN, -Infinity, Infinity, -Infinity, 0.0, 0.0, 2.107166861527933E307, -Infinity, 3.0436493463409975E307, NaN, 6.161752096997708E307, Infinity, 1.573297551247543E308]
[2.381551804921292E307, 1.4160379253299807E308, NaN, 1.746175578886582E308, 5.1755703872016E307, 0.0, NaN, -Infinity, 1.0190345057871029E308, 2.0612149982396432E307, 4.9E-324, -Infinity, 4.9E-324, 0.0, 1.0194962943671548E308, 0.0, Infinity, 5.077528176976716E307, NaN, 0.0, 1.3268066926276577E308, 4.9E-324, -Infinity, 5.150500222502397E307, NaN, NaN, NaN, Infinity, 1.5794104203790765E308, 8.880968036574578E307, 1.5641818724565498E308, NaN, 1.7821457737206092E308, 5.4394359809471256E306, 1.1769748213979329E308, -Infinity, NaN, 0.0, 1.6303420858523774E308, 0.0, NaN, -Infinity, 4.9E-324, 4.9E-324, 4.9E-324, NaN, -Infinity, NaN, NaN, NaN]
 ```

Reply:
try simple values, as it may return `pi's` or some multiple of it or nan's with infinity/nan's and negative values.

```
x: 7, 1, 2, 10
y: 4, 10, 8, 2
```

Reply:
I have added test for these values as well.